denonavr/denonavr.py

```diff
@@ -469,10 +469,7 @@ def ensure_configuration(self):
             if self.manufacturer is None and self.model_name is None:
                 executor.submit(self.get_device_info)
 
-            if self._receiver_type == AVR_X_2016.type:
-                executor.submit(self._get_zone_name)
-            else:
-                executor.submit(self._get_receiver_name)
+            executor.submit(self._get_receiver_name)
```

Comment:
This removing functionality rather then a fix.
In case of multiple zone each zone would have the same name.

Reply:
see my comment below

Reply:
have reverted this in my fork